### DISCUSSION TITLE
Tag for warning about known issues (CVE, critical issues, ...)

### DISCUSSION THREAD

**Discussion by cobratbq** [s4u/pgpverify-maven-plugin#350] [Ideas]
I am wondering whether or not it would be helpful to allow for an additional tag (apart from `noKey`, `noSig`, etc.) to warn of a different kind of concern. That is, the plug-in now warns for any deviations according to identity, authenticity and revocation. I single additional tag that allows for a warning text could be used to warn for known critical issues, like the recent [log4j issue](https://nvd.nist.gov/vuln/detail/CVE-2021-44228). Most of the infrastructure is in place. We would only need a tag with an ability to display a custom message.

Arguably, this can be considered widening the scope, even though there is infrastructure to revoke identities -- but not individual packages -- based on concerns. Therefore posting it as an idea only.

**Comment by slawekjaranowski** [comment]
I don't think that will be good idea. 
It is totally different responsibility.
There are many plugins which check security vulnerability and manage global database.

Example:
https://sonatype.github.io/ossindex-maven/maven-plugin/
https://owasp.org/www-project-dependency-check/

**Reply by cobratbq** [reply]
right, I expected that answer. Thanks for the alternative options.
